app/views/core/CreateAccountModal/CreateAccountModal.coffee

```diff
@@ -0,0 +1,104 @@
+BasicInfoView = require './BasicInfoView'
+SingleSignOnAlreadyExistsView = require './SingleSignOnAlreadyExistsView'
+SingleSignOnConfirmView = require './SingleSignOnConfirmView'
+ConfirmationView = require './ConfirmationView'
+State = require 'models/State'
+template = require 'templates/core/create-account-modal/create-account-modal'
+forms = require 'core/forms'
+User = require 'models/User'
+application  = require 'core/application'
+errors = require 'core/errors'
+utils = require 'core/utils'
+
+###
+CreateAccountModal is a wizard-style modal with several subviews, one for each
+`screen` that the user navigates forward and back through.
+
+There are three `path`s, one for each account type (individual, student).
+Teacher account path will be added later; for now it defers to /teachers/signup)
+Each subview handles only one `screen`, but all three `path` variants because
+their logic is largely the same.
+
+They `screen`s are:
+  choose-account-type: Sets the `path`.
+  segment-check: Checks required info for the path (age, )
+    coppa-deny: Seen if the indidual segment-check age is < 13 years old
+  basic-info: This is the form for username/password/email/etc.
+              It asks for whatever is needed for this type of user.
+              It also handles the actual user creation.
+              A user may create their account here, or connect with facebook/g+
+    sso-confirm: Alternate version of basic-info for new facebook/g+ users
+  sso-already-exists: When facebook/g+ user already exists, this prompts them to sign in.
+  extras: Not yet implemented
+  confirmation: Not yet implemented
+
+NOTE: BasicInfoView's two children (SingleSignOn...View) inherit from it.
+This allows them to have the same form-handling logic, but different templates.
+###
+
+module.exports = class CreateAccountModal extends ModalView
+  id: 'create-account-modal'
+  template: template
+  closesOnClickOutside: false
+  retainSubviews: true
+
+  events:
+    'click .login-link': 'onClickLoginLink'
+    'click .back-to-segment-check': -> @state.set { screen: 'segment-check' }
+
+  initialize: (options={}) ->
+    classCode = utils.getQueryVariable('_cc', undefined)
+    @signupState = new State {
+      path: if classCode then 'student' else null
+      screen: if classCode then 'segment-check' else 'choose-account-type'
+      ssoUsed: null # or 'facebook', 'gplus'
+      classroom: null # or Classroom instance
+      facebookEnabled: application.facebookHandler.apiLoaded
+      gplusEnabled: application.gplusHandler.apiLoaded
+      classCode
+      birthday: new Date('') # so that birthday.getTime() is NaN
+    }
+
+    @listenTo @signupState, 'all', @render #TODO: debounce
+
+    @listenTo @insertSubView(new ChooseAccountTypeView()),
+      'choose-path': (path) ->
+        if path is 'teacher'
+          application.router.navigate('/teachers/signup', trigger: true)
+        else
+          @signupState.set { path, screen: 'segment-check' }
```

Comment:
Neither did I! It doesn't really say it in the docs. But I wanted it to be able to, tried it, and was happy when it worked...